src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -433,12 +433,19 @@ void
 OverlayImpl::setupValidatorKeyManifests (BasicConfig const& config,
                                          DatabaseCon& db)
 {
-    auto const loaded = manifestCache_.loadValidatorKeys (
+    auto loaded = manifestCache_.loadValidatorKeys (
+        config.section ("validators"),
+        journal_);
+
+    if (!loaded)
+        Throw<std::runtime_error> ("Unable to load keys from [validators]");
+
```

Comment:
These strings are short enough to fit into all known short-string buffers.  Unfortunately we don't use the short-string-optimization on gcc (`_GLIBCXX_USE_CXX11_ABII == 0`).